README.md

```diff
@@ -438,26 +481,52 @@ declaring class. An inherited annotated method will NOT be recognized
 and used! To use inheritance, declare another annotated method that
 delegates the call, such as
 
-	@Before def setup { super.setup }
+```scala
+@Before def setup { super.setup }
+```
 
---- Field annotations ---
+## Field annotations
 
-Modbat currently supports one model field annotation, @Trace.
+Modbat currently supports one model field annotation, `@Trace`.
 Model fields with this annotation are traced throughout test execution.
 After each step, all annotated fields of the model whose transition
 just executed, is checked for updates of these fields. Updates are
 logged and shown as part of the error trace.
 
---- Visualization ---
+## Visualization
 
 The tool supports a basic visualization (requiring graphviz), using
 
 	scala modbat.jar --mode=dot <model>
 
-The output file is <modelname>.dot. The destination directory can
+The output file is `<modelname>.dot`. The destination directory can
 be changed using --dot-dir=...; default is the current directory.
 
---- How to compile your own model ---
+Example:
+
+    modbat/build$ scala modbat.jar --classpath=modbat-examples.jar \
+                                   --mode=dot \
+                                   -n=1 -s=2455cfedeadbeef \
+                                   --no-redirect-out \
+                                   modbat.examples.SimpleModel
+    modbat/build$ cat modbat.examples.SimpleModel.dot
+    digraph model {
+      orientation = landscape;
+      graph [ rankdir = "TB", ranksep="0.4", nodesep="0.2" ];
+      node [ fontname = "Helvetica", fontsize="12.0", margin="0.07" ];
+      edge [ fontname = "Helvetica", fontsize="12.0", margin="0.05" ];
+      "" [ shape = "point", height="0.1" ];
+      "" -> reset
+      reset -> zero [ label = " new SimpleCounter " ];
+      zero  -> zero [ label = " toggleSwitch " ];
+      zero  -> one [ label = " inc " ];
+      one   -> two [ label = " inc " ];
+      zero  -> two [ label = " inc2 " ];
+      two   -> end [ label = " assert " ];
+    }
```

Comment:
@cyrille-artho I think I'll leave this as it is though since it's more of an example that you're not supposed to copy-paste.